.travis.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.6"
+
+before_install:
+  - wget -O requirements.txt https://dockstore.org/api/metadata/runner_dependencies?client_version=1.6.0&python_version=3
```

Comment:
What happens when Dockstore goes to 1.7 on dockstore.org? Do you have to remember to update this, or will it continue to work?

Reply:
Can probably just go to the cwl1.1 dependencies since it should run 1.0 fine and grab requirements.txt from github like [Denis mentioned](https://github.com/CancerCollaboratory/dockstore-tool-bamstats/pull/10#issuecomment-532361688) in another PR.

> https://raw.githubusercontent.com/dockstore/dockstore/1.7.0-rc.2/dockstore-webservice/src/main/resources/requirements/1.7.0/requirements3.txt